rpm.go

```diff
@@ -136,14 +154,16 @@ func NewRPM(m RPMMetaData) (*RPM, error) {
 	m.Compressor = compressorName
 
 	rpm := &RPM{
-		RPMMetaData:       m,
-		di:                newDirIndex(),
-		payload:           p,
-		compressedPayload: z,
-		cpio:              cpio.NewWriter(z),
-		files:             make(map[string]RPMFile),
-		customTags:        make(map[int]IndexEntry),
-		customSigs:        make(map[int]IndexEntry),
+		RPMMetaData:          m,
+		di:                   newDirIndex(),
+		payload:              p,
+		compressedPayload:    z,
+		cpio:                 cpio.NewWriter(z),
+		files:                make(map[string]RPMFile),
+		customTags:           make(map[int]IndexEntry),
+		customSigs:           make(map[int]IndexEntry),
+		scriptletInterpreter: DefaultScriptletInterpreter,
```

Comment:
`scriptletInterpreter` can be removed, it always references `DefaultScriptletInterpreter` now.